src/index.ts

```diff
@@ -160,7 +167,7 @@ async function uploadProfile(profile: perftools.perftools.profiles.IProfile) {
 
     const url = `${config.serverAddress}/ingest?name=${encodeURIComponent(
       config.appName
-    )}{${tagList}}&sampleRate=${SAMPLERATE}&spyName=nodespy`
+    )}{${tagList}}&sampleRate=${SAMPLING_DURATION}&spyName=nodespy`
```

Comment:
Pyroscope API expects sampleRate to be in Hz. I think it should be `1000 / SAMPLE_RATE (in ms)`

Reply:
got it! Thanks for pointing it out.